### ISSUE TITLE
won  game , no tokens for the winning

### ISSUE THREAD

**Issue by euroxen** [ValveSoftware/Dota2-Gameplay#20781]
### Description

won  game , no tokens for the winning

7887469302

### Example Match ID (and possibly Timestamp)

7887469302

**Comment by MonyThor** [comment]
Yes, I agree, I played three games in the second act, but it didn’t give tokens

**Comment by MonyThor** [comment]
Yes, I agree, I played two games in the second act, but it didn’t give tokens for the winning game
Match ID: 7887528796 / 7887333761
